examples/website/trips/app.js

```diff
@@ -99,7 +99,7 @@ export class App extends Component {
         data: buildings,
         extruded: true,
         wireframe: false,
-        fp64: true,
+        fp64: false,
```

Comment:
any specific reason?

Reply:
we don't need specify fp64 now, there is an automatic switch in zoom 12, from regular mode to offset mode. With fp64 flag, there is a warning printed.